src/commands/indexer/index.ts

```diff
@@ -151,8 +175,9 @@ export default class Indexer extends HealthCheck {
       warp: flags.warp,
     })
 
+    // TODO: It doesn't seems like this sync is working
     // Indexer always synchronizes missed blocks
-    this.networkMonitor.latestBlockHeight = await this.networkMonitor.loadLastBlocks(this.config.configDir)
+    // this.networkMonitor.latestBlockHeight = await this.networkMonitor.loadLastBlocks(this.config.configDir)
```

Comment:
Maybe we want to reenable this or just leave it disabled. Not totally sure what to do about it, but probably needs more debugging to figure out why sync doesn't seem to work as expected which could be a time suck right now